containers/hydra/messager.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i python3 -p "python3.withPackages(ps: [ ps.slackclient ])"
+#
+# SPDX-FileCopyrightText: 2023 Technology Innovation Institute (TII)
+# SPDX-License-Identifier: Apache-2.0
+# ------------------------------------------------------------------------
+# Hydra Slack messaging script
+#
+# Prints given messaage to configured Slack channel with major hydra build
+# information data
+#
+# Slack configuration file format:
+# line1: Slack auth secret token (given when Slack application created)
+# line2: Used Slack channel (Slack application must be installed for this channel for usage)
+#
+# Do not keep this auth file in version control system. Postbuild (user of this script) assumes
+# it is stored to ci-public/containers/hydra/store/home in host side.
```

Comment:
Not true. There's no assumption about store being in a specific place on the host side - in a typical production configuration it's **not** in the path documented above.